core/templates/dev/head/components/answerGroupEditor.js

```diff
@@ -153,6 +153,8 @@ oppia.directive('answerGroupEditor', [function() {
           return [getDefaultInputValue('Real'), getDefaultInputValue('Real')];
         case 'ListOfUnicodeString':
         case 'SetOfUnicodeString':
+        case 'SetOfHtmlString':
+          return false;
```

Comment:
This seems wrong to me. The default value should be an empty list, right? In which case you don't need this line.